### PR TITLE
Implemented the EventsLog tab (among other things)

### PR DESCRIPTION
**Pull request by amitnovick**
Feel free to raise questions / report issues with anything included here :-)

### CODE REVIEW

File: packages/xstate-dev/panel/package.json [hunk 1]
```diff
@@ -7,17 +7,29 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "@xstate/react": "^0.8.1",
+    "date-fns": "^2.9.0",
+    "hex-rgba": "^1.0.2",
+    "immutable": "^4.0.0-rc.12",
```
Comment (davidkpiano) [inline packages/xstate-dev/panel/package.json]: What is `immutable` being used for? Can we please avoid it?

File: packages/xstate-dev/panel/src/utils/getItemString.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Iterable } from 'immutable';
```
Comment (davidkpiano) [inline packages/xstate-dev/panel/src/utils/getItemString.js]: I don't think we should support `immutable`.

**Comment by davidkpiano** [comment]
It looks good to me, except for `immutable` support. We should absolutely not support `immutable`.

File: packages/xstate-dev/panel/src/utils/getItemString.js (hunk 2 above)
Comment (amitnovick) [reply packages/xstate-dev/panel/src/utils/getItemString.js]: Agree!
I am not sure what it's being used for (extracted it from elsewhere). 
Will try to remove it and update the PR

**Pull request merged**